### ISSUE TITLE
Objective 3 broken links and unclear/missing information

### ISSUE THREAD

**Issue by psonnera** [openaps/AndroidAPSdocs#2037]
(Gerrit - from Discord)

I finished the Objective 3 question set yesterday. Some of the URL links in those questions are broken, they go to 404 pages.
1. Noisy CGM Readings
2. Troubleshooting (only the facebook link)

Some of the questions also have URL's which are only vaguely relevant to the question, and some dont even have the information in them to help you learn the content to answer the question. 

There is that one about Sensitivity plugins. The wording used, cannot even be found in the wiki when doing a search. I later realized it was specifically about autosens, but the question title gives the impression there are many different plugins, and this was a general question about all of them. One of the answers was about Autotune, which tripped me up, but if I had known the question was about Autosens rather than 'Sensitivity plugins' then at least I would have known what the question was asking me.

**Comment by emilisev** [comment]
I don't think this issue is related to the web docs, but rather the links provided in AAPS itself when doing the objectives. The web doc for objective 3 does not provide any detail on the specific questions asked.
I don't reproduce the broken links in AAPS with the French version, but I remember from doing the objectives myself the same general impression of inappropriate doc and really hard to find the information regarding the questions. Perhaps an entirely different project go get started on ?!

**Comment by psonnera** [comment]
From @HellDuke in Discord: Hey quick question, maybe I missed something, but I noticed some discrepancies while doing Objective 3.

For one the question on Updating AAPS expects `When updated versions of AAPS are released, the earlier versions may be remotely limited after a specified time.` to be selected, however this does not seem to be mentioned anywhere in the documentation (at least I could not find mention of this). There are points advising to do the update as soon as possible, without giving any reason or indication of a remote limiting mechanism. A natural conclusion would be that you need to update as soon as possible due to bugs or potential security issues, but feature or complete application limitation (not even clear from the answer WHAT the limitation might be) is not on the forefront of peoples minds I think, which can lead to frustration when getting a wrong answer for not picking that one.

Then in the updates it also expects you to pick `You need to have Git installed and configured on your computer.` which is similar to the expectation in the Prerequisites question `A computer with Android Studio installed and configured.` yet the documentation flow makes these as one of 2 options. So while yes, you need these when doing the build on your PC, a entirely valid approach would have a user completely bypass that part of the documentation and have no use for it. Since the question asks for what is mandatory it's not unreasonable for a user to not select options that are optional.

Mabe I am overthinking this, but thought I'd just point it out because I feel like this could lead to a bit of unnecessary frustration. I know it did for me, making me question if I accidentally tapped a wrong checkbox a few times, leading to me just looking at the source code to verify the expected answers (which kind of defeats the purpose of objective 3 I think). So it just makes me wonder, did I miss something or are some questions just out of date?

**Comment by psonnera** [comment]
More data from Facebook to help understanding the issues encountered:

> I've been stuck in this question for 3 days, the last one in Objective3. 
> I know you cannot build an apx from a friend. I know the last two are correct - updated versions should be installed and then the old versions get to end of life and then one needs to save the keystroke and update using the same one. I am hesitating on the second option, you need Git but can also have in on the phone, not neccessarily on the PC(I installed it via my phone only). I tried with both fake and true for second option. I am going nuts...What am I judging wrong?

<img width="272" height="590" alt="Image" src="https://github.com/user-attachments/assets/2277a5b4-e6b7-419b-8623-1ece158ff151" />

**Comment by psonnera** [comment]
To be checked for V4, need AAPS side integration

**Comment by psonnera** [comment]
Moved from https://github.com/openaps/AndroidAPSdocs/issues/2212 - to be reviewed with items above

```
<string name="pumpdisconnect_label">Reasons for applying "Disconnect pump" in AAPS</string>
<string name="pumpdisconnect_whattodo">What should be done when disconnecting the pump from your body?</string>
<string name="pumpdisconnect_unnecessary">This is unnecessary as insulin will not be delivered if the pump is physically disconnected.</string>
<string name="pumpdisconnect_missinginsulin">It prevents AAPS from accounting for insulin that was not delivered whilst the pump is physically disconnected.</string>
<string name="pumpdisconnect_notstop">It will not stop insulin delivery if the pump remains connected to your body.</string>
<string name="pumpdisconnect_openloop">It will send AAPS into open loop mode.</string>
<string name="pumpdisconnect_hint1">https://wiki.aaps.app/en/latest/DailyLifeWithAaps/AapsScreens.html#loop-status</string>
```

**Comment by psonnera** [comment]
Objectives have been reworked and successfully checked against the documentation.
